src/SignalR/server/Core/src/HubOptions.cs

```diff
@@ -53,5 +53,25 @@ public class HubOptions
         public int? StreamBufferCapacity { get; set; } = null;
 
         internal List<IHubFilter>? HubFilters { get; set; }
+
+        /// <summary>
+        /// By default a client is only allowed to invoke a single Hub method at a time.
+        /// Changing this property will allow clients to invoke multiple methods at the same time before queueing.
+        /// </summary>
+        public int MaxParallelInvocationsPerClient
```

Comment:
```suggestion
        public int MaximumParallelInvocationsPerClient
```

Pranav's comment didn't go through